databuilder/extractor/dashboard/mode_analytics/mode_dashboard_utils.py

```diff
@@ -28,21 +28,28 @@ def get_spaces_query_api(conf: ConfigTree) -> BaseRestApiQuery:
         :return:
         """
 
-        # https://mode.com/developer/api-reference/management/spaces/#listSpaces
-        spaces_url_template = 'https://app.mode.com/api/{organization}/spaces?filter=all'
+        # https://mode.com/developer/discovery-api/analytics/spaces
+        spaces_url_template = 'https://app.mode.com/batch/{organization}/spaces'
 
         # Seed query record for next query api to join with
         seed_record = [{'organization': conf.get_string(ORGANIZATION)}]
         seed_query = RestApiQuerySeed(seed_record=seed_record)
 
-        # Spaces
-        params = {'auth': HTTPBasicAuth(conf.get_string(MODE_ACCESS_TOKEN),
-                                        conf.get_string(MODE_PASSWORD_TOKEN))}
+        # mode_bearer_token must be provided in the conf
```

Comment:
Is this different from the normal api token? If so, could you document it in the config?

Reply:
Mode provides two kind of authentication. discovery api requires bearer token while the main api requires user password. see https://github.com/amundsen-io/amundsendatabuilder/blob/c0557a5c768a647032bccfa24f2218bbc13b8c04/databuilder/extractor/dashboard/mode_analytics/mode_dashboard_utils.py#L57